pallets/qa-tools/src/pallet_tools/liquidity_proxy.rs

```diff
@@ -0,0 +1,56 @@
+pub mod source_initializers {
+    use crate::{settings, Config};
+    use codec::{Decode, Encode};
+    use common::prelude::BalanceUnit;
+    use frame_support::dispatch::DispatchResult;
+    use frame_system::pallet_prelude::BlockNumberFor;
+    use order_book::{MomentOf, OrderBookId};
+    use sp_std::fmt::Debug;
+    use sp_std::vec::Vec;
+
+    #[derive(Clone, PartialEq, Eq, Encode, Decode, scale_info::TypeInfo, Debug)]
+    #[scale_info(skip_type_params(T))]
+    pub struct XYKPair<DEXId, AssetId> {
```

Comment:
Where is it used?

Reply:
This was adding when working on another qa-tools task, so I accidentally brought it here :)